.github /ISSUE_TEMPLATE/otel_q_and_a.md

```diff
@@ -0,0 +1,31 @@
+---
+name: OTel Q&A
+about: Issue tracking a new episode of OTel Q&A
+title: 'OTel Q&A: <name>'
+labels: OTel Q&A
+assignees: ''
+
+---
+<!-- Please remember to change the title of this issue by replacing <name> with the actual name for the session. This will be the title used in event invites -->
+
+### Date and time for the session
+<!-- Add here the proposed date for the event after it's been agreed -->
+
+TBD
+
+### Description used for events
+<!-- Add here a description that will be used in events generated for the event -->
+
+### Tasks
+The following list contains sub-tasks that need to be completed as part of organising, recording, and publicising the episode. Please replace items with individual links to issues where appropriate.
+
+- [ ] Confirm speakers
+- [ ] Confirm date and time
+- [ ] Create Q&A doc
+- [ ] Create promo image for socials
+- [ ] Create OpenTelemetry public calendar event (including Zoom link)
+- [ ] Create CNCF event
+- [ ] Promote on OpenTelemetry socials (LinkedIn, X, Bluesky, Mastodon)
+- [ ] Promote on CNCF Slack channels ([#otel-announcements](https://cloud-native.slack.com/archives/C049G9W5NQ7), [#otel-sig-end-users](https://cloud-native.slack.com/archives/C01RT3MSWGZ), [#o11y-ambassadors](https://cloud-native.slack.com/archives/C050WLWHV1U))
+- [ ] Host session
+- [ ] Edit and publish recording
```

Comment:
```suggestion
- [ ] Edit recording
- [ ] Publish recording
```

Split these into 2 items since it may not be the same person completing these tasks